PluralKit.Bot/Commands/MemberEdit.cs

```diff
@@ -362,29 +360,127 @@ public async Task Privacy(Context ctx, PKMember target, PrivacyLevel? newValueFr
+            }
+            else if(ctx.Match("birthday", "birth", "bday"))
+            {
+                subjectStr = "birthday";
+                target.BirthdayPrivacy = PopPrivacyLevel("birthday", out levelStr, out levelExplanation);
+            }
+            else if(ctx.Match("pronouns", "pronoun"))
+            {
+                subjectStr = "pronouns";
+                target.PronounPrivacy = PopPrivacyLevel("pronouns", out levelStr, out levelExplanation);
+            }
+            else if(ctx.Match("color","colour"))
+            {
+                subjectStr = "color";
+                target.ColorPrivacy = PopPrivacyLevel("color", out levelStr, out levelExplanation);
+            }
+            else if(ctx.Match("created","datecreated","date"))
+            {
+                subjectStr = "date created";
+                target.CreatedTimestampPrivacy = PopPrivacyLevel("color", out levelStr, out levelExplanation);
+            }
+            else if(ctx.Match("messages","message","messagecount"))
+            {
+                subjectStr = "messages";
+                target.MessageCountPrivacy = PopPrivacyLevel("messages", out levelStr, out levelExplanation);
+            }
+            else if(ctx.Match("visibility","hidden","shown"))
+            {
+                subjectStr = "visibility";
+                target.MemberVisibility = PopPrivacyLevel("visibility", out levelStr, out levelExplanation);
+            }
+            else if(ctx.Match("all")){
+                subjectStr = "all";
+                PrivacyLevel level = PopPrivacyLevel("all", out levelStr, out levelExplanation);
+                target.MemberVisibility = level;
+                target.NamePrivacy = level;
+                target.DescriptionPrivacy = level;
+                target.BirthdayPrivacy = level;
+                target.PronounPrivacy = level;
+                target.ColorPrivacy = level;
+                target.CreatedTimestampPrivacy = level;
+                target.MessageCountPrivacy = level;
+            }            
+            else
+                throw new PKSyntaxError($"Invalid privacy subject `{ctx.PopArgument().SanitizeMentions()}` (must be {subjectList}).");
 
-            if (newValue == PrivacyLevel.Private)
-                await ctx.Reply($"{Emojis.Success} Member privacy set to **private**. This member will no longer show up in member lists and will return limited information when queried by other accounts.");
+
+            await _data.SaveMember(target);
+            //Handle "all" subject
+            if(subjectStr == "all"){
+                if(levelStr == "private")
+                    await ctx.Reply($"All {target.Name}'s privacy settings have been set to **{levelStr}**. Other accounts will now see nothing on the member card");
+                else 
+                    await ctx.Reply($"All {target.Name}'s privacy settings have been set to **{levelStr}**. Other accounts will now see everything on the member card");
```

Comment:
Nitpick: need periods at the end of these messages, as well as sanitization on target.Name.